### PR TITLE
Scatterplot/  tooltip

### PR DESCRIPTION
**Pull request by kzadurska**
Hover over point and render tooltip:

tooltip-within-stage

Zrzut ekranu 2022-02-16 o 11 26 23

### CODE REVIEW

File: src/client/visualizations/scatterplot/point.tsx [hunk 1]
```diff
@@ -27,19 +27,33 @@ interface PointProps {
   yScale: LinearScale;
   xSeries: ConcreteSeries;
   ySeries: ConcreteSeries;
+  onHover(datum: Datum): void;
```
Comment (adrianmroz-allegro) [inline src/client/visualizations/scatterplot/point.tsx:30]: Maybe `setHover` and `resetHover`? `Point` doesn't need to understand what does it mean that hover is null.

Note: We have very lenient tsconfig, but don't define function with specific type parameter and pass null.

Comment (kzadurska) [reply src/client/visualizations/scatterplot/point.tsx]: Maybe it's too lenient 🤔  Anyway, good point, resolved!

Comment (adrianmroz-allegro) [reply src/client/visualizations/scatterplot/point.tsx]: It is. But changing it to more strict resulted in one big scary pile of errors 😱

**Review by adrianmroz-allegro — APPROVED** [review]
We're getting there!

**Pull request merged**